soloconnect/settings.py

```diff
@@ -90,10 +92,19 @@
         "NAME": os.getenv("RDS_DB_NAME"),
         "PASSWORD": os.getenv("RDS_PASSWORD"),
         "ENGINE": "django.db.backends.postgresql",
-        "TEST": {"NAME": "testdatabase"},
+        "TEST": {"NAME": "testdatabase2", "ENGINE": "django.db.backends.sqlite3"},
     }
 }
-
+if "test" in sys.argv:
+    DATABASES["default"] = {
+        "ENGINE": "django.db.backends.postgresql_psycopg2",
+        "HOST": os.getenv("TEST_RDS_HOSTNAME"),
+        "USER": os.getenv("TEST_RDS_USERNAME"),
+        "PASSWORD": os.getenv("TEST_RDS_PASSWORD"),
+        "NAME": os.getenv("TEST_RDS_DB_NAME"),
+        "PORT": os.getenv("TEST_RDS_DB_PORT"),
+        "TEST": {"NAME": os.environ.get("TEST_DATABASE_NAME")},
+    }
```

Comment:
The implication of using a sqlite database as our test db is that we can't test pg-specific features, right?

Reply:
Yep